sage_modabvar/abvar_newform.py

```diff
@@ -235,3 +238,77 @@ def _calculate_endomorphism_generators(self):
 
         return V.saturation().basis()
 
+    def frobenius_polynomial(self, p):
+        """
+        Computes the frobius polynomial at `p`.
+
+        INPUT:
+
+        - ``p`` -- prime number
+
+        OUTPUT:
+            a rational polynomial
```

Comment:
It would be better to return a poly over ZZ and say the output is "a monic integral polynomial".